games-emulation/gw_libretro/gw_libretro-1.6.2_20180125.recipe

```diff
@@ -0,0 +1,54 @@
+SUMMARY="A libretro core for Game & Watch simulators"
+DESCRIPTION="gw-libretro is a libretro core that runs Game & Watch simulators. \
+It runs simulators converted from source code. The converted games are \
+available at libretro's buildbot site. Unpack the zip and read the README.txt \
+for instructions."
+HOMEPAGE="https://bot.libretro.com/assets/cores/Handheld%20Electronic%20Game/"
+COPYRIGHT="2013-2018 the libretro team"
+LICENSE="Zlib
+	Public Domain"	
```

Comment:
```diff
-	Public Domain"	
+	Public Domain"
```
(There's a trailing tab at the end of the line.)

Reply:
Whoops. Fixed.